mediaplayerbackend.cpp

```diff
@@ -208,31 +210,36 @@ void MediaPlayerBackend::setCurrentIndex(int index)
 
 void MediaPlayerBackend::setMuted(bool muted)
 {
+    qDebug() << "setMuted: " << muted;
     m_mixerController.setMute(muted);
 }
 
 void MediaPlayerBackend::setPlayMode(QIviMediaPlayer::PlayMode playMode)
 {
     switch (playMode) {
     case QIviMediaPlayer::Normal:
+        qDebug() << "setPlayMode: Normal";
         m_tracklistController.setSingle(false);
         m_tracklistController.setRepeat(false);
         m_tracklistController.setRandom(false);
         break;
 
     case QIviMediaPlayer::RepeatTrack:
+        qDebug() << "setPlayMode: RepeatTrack";
         m_tracklistController.setSingle(true);
         m_tracklistController.setRepeat(true);
         m_tracklistController.setRandom(false);
         break;
 
     case QIviMediaPlayer::RepeatAll:
+        qDebug() << "setPlayMode: RepeatAll";
         m_tracklistController.setSingle(false);
         m_tracklistController.setRepeat(true);
         m_tracklistController.setRandom(false);
         break;
 
     case QIviMediaPlayer::Shuffle:
+        qDebug() << "setPlayMode: Shuffle";
```

Comment:
pl add default case

Reply:
what would default case do?

Reply:
For example:
default:
  print qWarning() << "setPlayMode: error, playMode not supported: " << playMode;
  break;